sources/packages/backend/apps/db-migrations/src/migrations/1758153112291-AlterDisbValueAmtNotNullable.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AlterDisbValueAmtNotNullable1758153112291
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Delete-null-value-amount-records.sql",
+        "DisbursementValues",
+      ),
+    );
+    await queryRunner.query(
+      getSQLFileData(
+        "Alter-value-amount-not-nullable.sql",
+        "DisbursementValues",
+      ),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Alter-value-amount-nullable.sql", "DisbursementValues"),
```

Comment:
The pattern to be followed for migrations up/down is to add a rollback sufix for the up command.
For instance, for `Alter-value-amount-not-nullable.sql` the down file should be named as `Rollback-alter-value-amount-not-nullable.sql`

Reply:
Fixed.